.gitignore

```diff
@@ -17,3 +17,4 @@
 # Configuration files
 .powenv
 .env
+config/database.yml
```

Comment:
As noted back on [#17 (more reasoning there)](https://github.com/edgi-govdata-archiving/web-monitoring-db/issues/17#issuecomment-307125039), I don’t think we should do this.